{{ cookiecutter.project_shortname }}/CONTRIBUTING.rst

```diff
@@ -0,0 +1,113 @@
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+Types of Contributions
+----------------------
```

Comment:
I think the CONTRIBUTING guide should be rather simple, kind of like the usual:

- https://github.com/inveniosoftware/claimstore/blob/master/CONTRIBUTING.rst
- https://github.com/cernopendata/opendata.cern.ch/blob/master/CONTRIBUTING.rst

and all the git etc details should rather point to the common documentation located on RTFD.  (coming with https://github.com/inveniosoftware/invenio/issues/1799)